pkg/webhooks/admission/jobs/validate/admit_job.go

```diff
@@ -229,6 +233,31 @@ func validateJobCreate(job *v1alpha1.Job, reviewResponse *admissionv1.AdmissionR
 	} else if queue.Status.State != schedulingv1beta1.QueueStateOpen {
 		msg += fmt.Sprintf(" can only submit job to queue with state `Open`, "+
 			"queue `%s` status is `%s`;", queue.Name, queue.Status.State)
+	} else if config.ConfigData != nil {
+		config.ConfigData.Lock()
+		enableHierarchyCapacity := config.ConfigData.EnableHierarchyCapacity
+		config.ConfigData.Unlock()
+		if enableHierarchyCapacity {
+			if queue.Name == "root" {
+				msg += fmt.Sprintf(" can not submit job to root queue;")
```

Comment:
In a scenario where there is only one root queue, can the root queue be considered a leaf queue and allocated jobs on it?

Reply:
Even if the user has not created any new queues, the system will still have two queues called "root" and "default". The "root" queue is not a leaf queue because the parent queue of "default" is "root".